src/xdocs/index.xml.vm

```diff
@@ -97,8 +97,25 @@
         <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
       </p>
       <p>
-        Checkstyle is not using Semantic Versioning
-        (<a href="https://github.com/checkstyle/checkstyle/issues/3709">reasons in issue #3709</a>)
+        Checkstyle is following
+        <a href="https://dafoster.net/articles/2015/03/14/semantic-versioning-vs-romantic-versioning/">
+          Hybrid Romantic and Semantic Versioning</a>:
+        <b>First</b>.<b>Second</b>.<b>Third</b>
+        <br/>
+        <b>First</b> First digit is representing Romantic version.
+           It means that some very noticable
+           (from maintainer's perspective) breaking compatibility happened
+           or major conceptual change<br/>
+        <b>Second</b> digit is Semantic version which is a combination of major and minor.
+           It means that either some breaking compatibility happened
+           or new features/modules were introduced.<br/>
```

Comment:
`It means that either some breaking compatibility happened or new features/modules were introduced` => `When it is the only number increasing, it means that either some minor breaking compatibility happened or new features/modules were introduced`

Reply:
done.
but without "minor" word in second part, as out braking changes are still painful, even recompilation might be required.

Reply:
I am good with "some breaking". I just wanted to distinguish between 1 and 2, large and more minor breaking.